experiments/grafana/docker-compose.yml

```diff
@@ -19,6 +19,15 @@ services:
     image: mongo:3.2
     ports:
       - "27017:27017"
+    deploy:
+      mode: global
```

Comment:
if this is to be used with docker compose, deploy configuration does not make sense.
if this is to be used with docker stack, this configuration, using global will create an instance on each node, but the instances won't be in synch